src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs

```diff
@@ -17174,5 +17174,135 @@ static unsafe void CheckNaN(float nan)
 ";
             var compilation = CompileAndVerify(source, options: TestOptions.ReleaseExe.WithAllowUnsafe(true), verify: Verification.Skipped, expectedOutput: @"");
         }
+
+        [WorkItem(47887, "https://github.com/dotnet/roslyn/issues/47887")]
+        [Fact]
+        public void ArrayAccessNInt()
```

Comment:
Consider moving these tests to `NativeIntegerTests`.

Reply:
Moved.